robot_calibration/src/optimizer.cpp

```diff
@@ -237,6 +238,56 @@ int Optimizer::optimize(OptimizationParams& params,
                                   NULL /* squared loss */,
                                   free_params);
       }
+      else if (params.error_blocks[j].type =="basecamera_to_ground")
```

Comment:
and this block will go away once freight_ground_error becomes part of ground_plane_error